spaces/boostedFinance/skin.scss

```
@@ -0,0 +1,10 @@
+.name {
```

Css selector should be similar to key
```suggestion
.boosted-finance {
```